common/bm_freertos.c

```diff
@@ -32,6 +32,16 @@ BmErr bm_queue_send(BmQueue queue, const void *item, uint32_t timeout_ms) {
   }
 }
 
+BmErr bm_queue_send_to_front_from_isr(BmQueue queue, const void *item) {
+  BaseType_t higher_priority_task_woken = pdFALSE;
+  if (xQueueSendToFrontFromISR(queue, item, &higher_priority_task_woken) == pdPASS) {
+    portYIELD_FROM_ISR(higher_priority_task_woken);
+    return BmOK;
+  } else {
+    return BmENOMEM;
+  }
```

Comment:
Would you be able to add that note from the comment description here as a comment in the code? That way if people decide to use it elsewhere, they will know how to use this API in a safe manner?